### DISCUSSION TITLE
[Feature] Offer option to transcode to same resolution as original source video

### DISCUSSION THREAD

**Discussion by dpnvektor** [immich-app/immich#2707] [Feature Request]
### The feature

It would be nice to give us an option to transcode into the same resolution as the source video, rather than forcing up or downscaling.  As it is, I don't want to loose resolution on a 4k video that I've recorded, and I don't want to waste space and CPU time by upscaling something recorded in 720 or 1080 from long ago into 4k.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mertalev** [comment]
That makes sense to me. I made a PR for it. #2709
